Hestia/app/src/main/java/hestia/UI/elements/DeviceBar.java

```diff
@@ -58,7 +62,8 @@ public void initView() {
         switc.setVisibility(View.INVISIBLE);
 
         for(final Activator activator : device.getActivators()){
-            if(activator.getRank() == 0){
+            int toggleRank = Integer.valueOf(context.getResources().getString(R.string.toggleRank));
```

Comment:
There is no pull request for that, so most likely, when it will be done, there will be merging conflicts anyway. At the same time, this branch has a very specific scope. I don't think we should worry about UI changes for now.